### PR TITLE
add foundant organization and community suite source

### PR DESCRIPTION
**Pull request by chrissuttonfoundant**

### CODE REVIEW

**Comment by chrissuttonfoundant** [comment]
I signed it!

File: sources/communitysuite.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "CSUITE",
+  "name": "CommunitySuite",
+  "categories": ["FINANCE"],
+  "organization": "FOUNDANT",
+  "iconUrl": "https://communityfoundations.foundant.com/wp-content/themes/genesis-ft-cf/images/foundant-community-foundations-40.gif",
+  "sourceUrl": "https://api.fcsuite.com/",
```
Comment (pfrisella) [inline sources/communitysuite.json]: Review instructions and description for sourceUrl at at https://github.com/googledatastudio/ds-data-registry#add-a-source

sourceUrl should point to a location users can learn more about the source of data.

File: sources/communitysuite.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "CSUITE",
```
Comment (pfrisella) [inline sources/communitysuite.json]: id should match file name with underscores for spaces, if applicable, e.g.
```suggestion
  "id": "COMMUNITYSUITE",
```

**Comment by pfrisella** [comment]
Friendly ping.
I'll wait a few days but if this is no longer required I will close.

**Comment by chrissuttonfoundant** [comment]
Sorry, I will close this pull request.  We will submit another one in a few weeks once we regroup on some things.

**Pull request closed without merging**